core/commands/pin.go

```diff
@@ -462,25 +463,23 @@ var verifyPinCmd = &cmds.Command{
 		cmds.Text: func(res cmds.Response) (io.Reader, error) {
 			quiet, _, _ := res.Request().Option("quiet").Bool()
 
-			outChan, ok := res.Output().(<-chan interface{})
+			out, err := unwrapOutput(res.Output())
```

Comment:
this looks like its being changed to only emit a single output. Whats going on here?

Reply:
I would like to know what is going on also.

Reply:
The marshaller is called for every message pushed down the channel. I am not sure why the legacy layer is doing this but it prints out all the results.

Reply:
(this is one of the many reasons I've given up on fixing issues with the compat layer and have started just converting commands to use the new system).